logger/common_test.go

```diff
@@ -125,20 +176,18 @@ func TestSendLogs(t *testing.T) {
 			require.NoError(t, err)
 			defer os.Remove(tmpIOSource.Name())
 			var (
-				expectedSize int64
 				testPipe     bytes.Buffer
 			)
 			for _, logMessage := range tc.logMessages {
-				expectedSize += int64(len([]rune(logMessage)))
-				_, err := testPipe.WriteString(logMessage)
+				_, err := testPipe.WriteString(logMessage + "\n")
 				require.NoError(t, err)
 			}
 
 			// Create a tmp file that used to inside customized dummy Log function where the
 			// logger sends log messages to.
 			tmpDest, err := ioutil.TempFile(os.TempDir(), "")
 			require.NoError(t, err)
-			defer os.Remove(tmpDest.Name())
+			//defer os.Remove(tmpDest.Name())
```

Comment:
Just wanted to confirm that we don't need this line anymore. If it's not needed, let's just get rid of it.

Reply:
Sorry good catch on that.. that's from when I was debugging the tests... will fix